scripts/run_multi_model.py

```
@@ -0,0 +1,352 @@
+        env.pop("CUSTOM_API_KEY", None)
+
+    cmd = [
+        sys.executable, "-m", "pytest",
+        "tests/adversarial/",
+        "-v", "-s",
+        "--tb=short",
+        "--timeout=120",
+    ]
+
+    start = datetime.now(timezone.utc)
+    result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)